README.md

```diff
@@ -110,76 +109,35 @@ npm run test:watch
 ![a diagram illustrating the proposed schema for Chapter](data/schema.png)
 </details>
 
-## User stories so far
+## User Stories
 
-Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers, both of which are users.
+### MVP
+The [MVP user stories are shown in the MVP Project](https://github.com/freeCodeCamp/chapter/projects/1) kanban / cards and as [issues marked with "MVP"](https://github.com/freeCodeCamp/chapter/labels/MVP).
 
-### As a user
+### Post-MVP
+We are maintaining a list of [post-MVP user stories](https://github.com/freeCodeCamp/chapter/issues/84).
```

Comment:
@vkWeb this is such a small concern, but from a technical writing standpoint we should never have a single bullet point. If/when a second bullet is added then we can certainly add bullets.

Reply:
Sorry, I meant to say that the user stories were well out of sync with the other two MVP documents.  I don't see why we'd want to maintain the same information in 2 or 3 places unless someone is going to actively maintain all of the locations.

It's already enough work just to keep things up to date so my vote is for more links and less of a huge README that gets out of date every day.

Reply:
Similar to the user stories and roadmap bullets, the Quick Links I added initially in the CONTRIBUTING.md are out of date.

My feeling is many of the bullets we initially added to the README and CONTRIBUTING made sense only temporarily because there was less structure and places to point newcomers in the initial chaos.

If/as other pages in the Github project go into more details it's fair to consider linking out to those sections to avoid huge and/or out of date .md files.

Reply:
@allella Umm.. Yes you are right. 

Let's keep it that way then ;)